AssuredWorkloads/src/V1beta1/resources/assured_workloads_service_rest_client_config.php

```diff
@@ -37,17 +38,6 @@
                     ],
                 ],
             ],
-            'ListWorkloads' => [
```

Comment:
At first pass, it doesn't seem like GetWorkload/ListWorkloads should be omitted from here. Need to investigate.

Reply:
@noahdietz any idea why these bindings were removed? I could not find anything in the CL history which reflects this change